_automating-configurations/security.md

```diff
@@ -4,31 +4,31 @@ title: Workflow security
 nav_order: 25
 ---
 
-# Flow Framework Security
+# Workflow security
```

Comment:
@dbwiddis Should we keep it as Workflow security or Flow Framework?

Reply:
The word "workflow" is commonly used in other places and has led to a lot of confusion.  So other than the API and documentation directly relating to it, I'd like to avoid using "workflow".

That said, the left-column header for our plugin is "Automating Configurations" and it seems that may be appropriate as well, although fitting that together with "security" seems awkward.

TLDR: I trust doc writers to pick here. I'm against "Workflow security" and prefer "Flow Framework" if that's the only option, but am very open to other headers.

Reply:
I'm equally not sure about the best heading for this topic. I would go with a consistent title "Automating workflows", "Workflow APIs", "Workflow security", etc. But we're already using "Automating configurations" and then everything else mentions "workflows". Adding "Flow Framework" might make the user confused about what we're referencing. That said, I see your point about workflow being a generic concept so I reverted to "Flow Framework security" and added the first sentence that explains that workflows are provided by the Flow Framework plugin. Maybe @natebower can weigh in on this during editorial review.

Reply:
I like that! Let me change to "workflow template security"

Reply:
@owaiskazi19 @dbwiddis I changed the topic name to "Workflow template security".